index.html

```diff
@@ -8,26 +8,62 @@
   <meta name="author" content="Wolox">
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
-<body class="container">
-  <div class="container-item badge">
-    <img class="book-cover" src="resources/book-cover.png" alt="book_cover"/>
-  </div>
-  <div class="container-item container-data">
-    <h2 class="bold-title border-line">Título del libro
-      <span class="base-span subtitle">(género)</span>
-    </h2>
-    <div class="item">
-      <span class="bold-text">Autor del libro:</span>
-      <span class="base-span">Nombre del autor del libro</span>
-    </div>
-    <div class="item">
-      <span class="bold-text">Editorial:</span>
-      <span class="base-span">Nombre de la Editorial</span>
-    </div>
-    <div class="item">
-      <span class="bold-text">Año de publicación:</span>
-      <span class="base-span">Año de publicación</span>
-    </div>
-  </div>
+<body class="list-container">
+  <a href="detail.html">
```

Comment:
After @beto @yo decided to use only the <a href="detail.html"> in the image, what do you think?

Reply:
it's ok, no problem I make these changes